src/platform/CMakeLists.txt

```diff
@@ -18,9 +18,9 @@ project(osconfig-platform)
 set(osconfig_platform_files
     ./Log.c
     ./Main.c
-    ./ManagementModule.cpp
-    ./ModulesManager.cpp
-    ./MpiServer.c)
+    ./Module.c
+    ./Platform.c
+    ./Server.c)

```

Comment:
>Server [](http://example.com/codeflow?start=6&length=6)

Rename these back to MpiServer.h | .c. This is what they contain. Not a generic or complete "server".